tools/sentieon_DNAscope_LongRead_CLI.cwl

```diff
@@ -0,0 +1,229 @@
+      prefix: -b
+    sbg:fileTypes: BED
+  haploid_bed:
+    type: File?
+    inputBinding:
+      position: 10
+      prefix: --haploid-bed
+    sbg:fileTypes: BED
+  gvcf:
+    type: boolean?
+    default: false
+    inputBinding:
+      position: 11
+      prefix: --gvcf
+  bam_format:
+    type: boolean?
+    default: false
+    doc: "Use the BAM format instead of CRAM for output aligned files (default: False)"
+    inputBinding:
+      position: 12
+      prefix: --bam_format
+  cores:
+    type: int?
+    doc: "Number of threads/processes to use"
+    inputBinding:
+      position: 13
+      prefix: -t
+  skip-small-variants:
+    type: boolean?
+    default: false
+    doc: "Skip small variant (SNV/indel) calling (default: False)"
+    inputBinding:
+      position: 14
+      prefix: --skip-small-variants
+  skip-svs:
+    type: boolean?
+    default: false
+    doc: "Skip SV calling (default: False)"
+    inputBinding:
+      position: 15
+      prefix: --skip-svs
+  skip-mosdepth:
+    type: boolean?
+    default: false
+    doc: "Skip QC with mosdepth (default: False)"
+    inputBinding:
+      position: 16
+      prefix: --skip-mosdepth
+  input_ref:
+    type: File?
+    doc: "Used to decode the input alignment file. Required if the input file is in the CRAM/uCRAM formats"
+    inputBinding:
+      position: 17
+      prefix: --input_ref
+    secondaryFiles:
+    - pattern: .fai
+      required: true
+  fastq_taglist:
+    type: string?
+    doc: "A comma-separated list of tags to retain. Defaults to ''*'' and the 'RG' tag is required"
+    inputBinding:
+      position: 18
+      prefix: --fastq_taglist
+  minimap2_args:
+    type: string?
+    doc: "Extra arguments for sentieon minimap2 (default: '-Y')"
+    inputBinding:
+      position: 20
+      prefix: --minimap2_args
+  util_sort_args:
+    type: string?
+    doc: "Extra arguments for sentieon util sort (default: '--cram_write_options version=3.0,compressor=rans')"
+    inputBinding:
+      position: 21
+      prefix: --util_sort_args
+  output_vcf:
+    type: string
+    doc: "Output VCF File. The file name must end in .vcf.gz"
+    inputBinding:
+      position: 100
+  cpu_per_job:
+    label: CPU per job
+    doc: CPU per job
+    type: int?
+    default: 36
+  mem_per_job:
+    label: Memory per job
+    doc: Memory per job[GB].
+    type: int?
+    default: 71
+
+outputs:
+  small_variants:
+    type: File
+    secondaryFiles:
+    - pattern: .tbi
+      required: true
+    outputBinding:
+      glob: $(inputs.output_vcf)
+    sbg:fileTypes: VCF.GZ
+  structural_variants:
+    type: File
+    secondaryFiles:
+    - pattern: .tbi
+      required: true
+    outputBinding:
+      glob: $(inputs.output_vcf.replace(".vcf.gz", ".sv.vcf.gz"))
+    sbg:fileTypes: VCF.GZ
+  out_alignments:
+    type: File[]?
+    secondaryFiles:
+    - pattern: .bai
+      required: false
+    - pattern: .crai
+      required: false
+    outputBinding:
+      glob: ["*.cram", "*.bam"]
```

Comment:
```suggestion
      glob: "*.[b|cr]am"
```
this works too, right?

Reply:
No. Haodong is leveraging the fact that CWL's can take an array of strings to pass to glob. What you're probably thinking about is `*.{b,cr}am` which would be a singular string that can grab both BAMs and CRAMs. Both solutions are acceptable.